pkgs/by-name/pl/plymouth-matrix-theme/package.nix

```diff
@@ -0,0 +1,48 @@
+{
+  stdenvNoCC,
+  fetchFromGitHub,
+  lib,
+  unstableGitUpdater,
+}:
+
+stdenvNoCC.mkDerivation {
+  pname = "plymouth-matrix-theme";
+  version = "0.1.0-unstable-19-02-2017";
+
+  src = fetchFromGitHub {
+    owner = "storax";
+    repo = "plymouth-matrix-theme";
+    rev = "b2268f25dea7537ed5709b00d5a83b3600265c54";
+    hash = "sha256-JmMmpw1By5U6OTaSPnJOZZxrieSnXivMmdt/JPazjpI=";
+  };
+
+  postPatch = ''
+    # Remove not needed files
+    rm README.rst LICENSE Makefile
+  '';
+
+  dontBuild = true;
+
+  installPhase = ''
+    runHook preInstall
+    mkdir -p $out/share/plymouth/themes/matrix
+    cp * $out/share/plymouth/themes/matrix
+    find $out/share/plymouth/themes/ -name \*.plymouth -exec sed -i "s@\/usr\/@$out\/@" {} \;
+    runHook postInstall
+  '';
+
+  passthru.updateScript = unstableGitUpdater;
```

Comment:
```
error:
       … while calling the 'derivationStrict' builtin

         at /builtin/derivation.nix:9:12: (source not available)

       … while evaluating derivation 'nixpkgs-update-script'
         whose name attribute is located at /home/puna/Development/nixpkgs2/pkgs/stdenv/generic/make-derivation.nix:331:7

       … while evaluating attribute 'shellHook' of derivation 'nixpkgs-update-script'

         at /home/puna/Development/nixpkgs2/maintainers/scripts/update.nix:223:3:

          222|   '';
          223|   shellHook = ''
             |   ^
          224|     unset shellHook # do not contaminate nested shells

       (stack trace truncated; use '--show-trace' to show the full trace)

       error: cannot coerce a set to a string
```

Should be
```suggestion
  passthru.updateScript = unstableGitUpdater { };
```

Reply:
Ahh. Can you or @superherointj make a PR following this? I am not on my system at the moment.